lib/include/tick/base/debug.h

```diff
@@ -40,6 +40,20 @@
 #define TICK_DEBUG_VERBOSE_MODE 1
 #endif
 
+#define TICK_CMP_REPORT(ss, v)                                         \
```

Comment:
Can we change variable names :)? And maybe add some comments to remember how it works in 6 months!